make/go.mk

```
@@ -74,7 +74,7 @@ bench.%: ## Runs go benchmarks for a specific module
 bench.%: ; $(info $(M) running benchmarks$(if $(GOTEST_RUN), matching "$(GOTEST_RUN)") for $* module)
 	$(if $(filter-out root,$*),cd $* && )go test $(if $(GOTEST_RUN),-run "$(GOTEST_RUN)") -race -cover -v ./...
 
-E2E_PARALLEL_NODES ?= $(shell nproc --ignore=1)
+E2E_PARALLEL_NODES ?= $(shell nproc)
```

This was originally here to not lock up local dev machines by using all processors. I wonder if we can change this to just use all procs only in CI?
```suggestion
ifeq ($(CI),true)
  E2E_PARALLEL_NODES ?= $(shell nproc)
else
  E2E_PARALLEL_NODES ?= $(shell nproc --ignore 1)
endif
```

---

Ran without this change locally and it all seems fine. I guess normal usage locally will focus tests so it will be ok to run utilising all CPUs. Let's try it out and see.